src/main/java/duke/Parser.java

```diff
@@ -0,0 +1,149 @@
+package duke;
+
+import duke.commands.AddDeadlineCommand;
+import duke.commands.AddEventCommand;
+import duke.commands.AddTodoCommand;
+import duke.commands.Command;
+import duke.commands.DeleteCommand;
+import duke.commands.ExitCommand;
+import duke.commands.FindCommand;
+import duke.commands.InvalidCommand;
+import duke.commands.ListCommand;
+import duke.commands.MarkCommand;
+
+import duke.exceptions.NotEnoughArgumentsException;
+import duke.exceptions.TaskNoDescriptionException;
+import duke.exceptions.DateNotInSequenceException;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+
+/**
+ * This is a class responsible for parsing the inputs given by user
```

Comment:
Perhaps put punctuation behind each parameter description. I have noticed this in other places too.